server/services/nl.py

```diff
@@ -368,8 +368,18 @@ def _containedin_classification(self, prediction,
         "province": ContainedInPlaceType.PROVINCE,
         "town": ContainedInPlaceType.TOWN,
         "zip": ContainedInPlaceType.ZIP,
+        # Schools (only plurals).
```

Comment:
could you add the note about order preference here too (that is highest pref first)